dstep/main.d

```diff
@@ -59,8 +59,9 @@ auto parseCLI (string[] args)
         std.getopt.config.passThrough,
         std.getopt.config.caseSensitive,
         "output|o", &config.output,
-        "language|x", &parseLanguage,
         "objective-c", &forceObjectiveC,
+        "language|x", "Treat subsequent input files as having type <language>.",
```

Comment:
Hmm, I think it's ok as a temporary fix, but I would prefer it to be in `Configuration.d`, however I don't see simple solution to do that right now...